src/v2/Components/v2/ArtworkFilter/ArtworkFilters/ColorFilter.tsx

```diff
@@ -10,7 +10,7 @@ export const ColorFilter: FC<ColorFilterProps> = ({ expanded = false }) => {
   const filterContext = useArtworkFilterContext()
 
   const toggleColor = color => {
-    if (filterContext.filters.color === color) {
```

Comment:
Here and throughout the existing filters, have modify them to read from the correct version of filter state in order to render UI correctly.